am-utils-6.2 - fix logical not comparison in get_ldap_timestamp()

From: Ian Kent <raven@themaw.net>

Looks like the not in this comparison is not meant to be applied to *ts.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 amd/info_ldap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amd/info_ldap.c b/amd/info_ldap.c
index cfed6fd..ecfc210 100644
--- a/amd/info_ldap.c
+++ b/amd/info_ldap.c
@@ -446,7 +446,7 @@ get_ldap_timestamp(ALD *a, char *map, time_t *ts)
 	   vals[0], map);
       err = ENOENT;
     }
-    if (!*ts > 0) {
+    if (!(*ts > 0)) {
       plog(XLOG_USER, "Nonpositive timestamp %ld for map %s\n",
 	   (u_long) *ts, map);
       err = ENOENT;
